pallets/pooled-staking/src/lib.rs

```diff
@@ -440,16 +449,20 @@ pub mod pallet {
             Calls::<T>::request_delegate(candidate, delegator, pool, stake)
         }
 
+        #[pallet::weight(T::WeightInfo::execute_pending_operations())]
```

Comment:
Weight doesn't depend of the number of operations in the Vec?

Reply:
fixed